pennylane/devices/experimental/default_qubit_2.py

```diff
@@ -45,12 +45,13 @@ class DefaultQubit2(Device):
     """A PennyLane device written in Python and capable of backpropagation derivatives.
 
     Keyword Args:
+         shots (int, Tuple[int], Tuple[Tuple[int]]): The default number of shots to use in executions involving
```

Comment:
Could `List` also be a valid input here? I ask because the types here are different to https://docs.pennylane.ai/en/stable/code/api/pennylane.QubitDevice.html

Reply:
We could probably switch to `Sequence` here.